src/ray/core_worker/experimental_mutable_object_provider.cc

```diff
@@ -177,14 +183,19 @@ void MutableObjectProvider::PollWriterClosure(
       object->GetData()->Size(),
       object->GetMetadata()->Size(),
```

Comment:
> `raylet_client_factory_`: Creates a function for each object. This function waits for changes on the object and then sends those changes to a remote node via RPC.

The above comment is from `experimental_mutable_object_provider.h`. Can you explain how does `reader->PushMutableObject` "sends those changes to a remote node via RPC"? Thanks!

Reply:
I fixed the comment.